### PR TITLE
Use pillar-cards in content and richlinks stylesheets if switched on

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- the pillar card swtich enables them everywhere, not just facia
- update facia switch to use the same method

### CODE REVIEW

File: admin/app/views/football/main.scala.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 @import controllers.admin.routes.UncachedWebAssets
 @import play.api.Mode.Dev
 @import conf.Static
+@import html.HtmlPageHelpers.pillarCardCSSFileContent
```
Comment (sndrs) [inline admin/app/views/football/main.scala.html]: @TBonnin this import doens't work but im not sure why – guess it's not available to templates. any idea how to accomplish this?

same with onward/app/views/richLink.scala.html

File: common/app/html/HtmlPage.scala [hunk 2]
```diff
@@ -41,4 +42,8 @@ object HtmlPageHelpers {
     )
   }
 
+  def pillarCardCSSFileFacia: String = if (PillarCards.isSwitchedOn) "facia.pc" else "facia"
+  def pillarCardCSSFileContent: String = if (PillarCards.isSwitchedOn) "content.pc" else "content"
+  def pillarCardCSSFileRichLinks: String = if (PillarCards.isSwitchedOn) "rich-links.pc" else "rich-links"
```
Comment (TBonnin) [inline common/app/html/HtmlPage.scala]: Not a big deal but I would keep the name shorter
For example:
```
FaciaCssFile
ContentCssFile
RichLinksCssFile
```

File: admin/app/views/football/main.scala.html (hunk 1 above)
Comment (TBonnin) [reply admin/app/views/football/main.scala.html]: Looks like it works. What error are you seeing?

**Review by TBonnin — APPROVED** [review]

File: admin/app/views/football/main.scala.html (hunk 1 above)
Comment (sndrs) [reply admin/app/views/football/main.scala.html]: hmm weird, it was erroring running `compile` last night, but seems ok now. maybe your reply was enough to fix it!

File: common/app/html/HtmlPage.scala [hunk 3]
```diff
@@ -19,7 +20,7 @@ object HtmlPageHelpers {
     def when(condition: => Boolean): Html = if(condition) html else Html("")
   }
 
-  def guardianHeaderHtml()(implicit page: model.Page, request: RequestHeader, applicationContext: ApplicationContext): Html = {
+  def guardianHeaderHtml()(implicit page: model.Page, request: RequestÍHeader, applicationContext: ApplicationContext): Html = {
```
Comment (NataliaLKB) [inline common/app/html/HtmlPage.scala]: `RequestÍHeader` ?

**Review by NataliaLKB — APPROVED** [review]
Other thank the `RequestÍHeader` comment, Everything else looks good!

File: common/app/html/HtmlPage.scala (hunk 3 above)
Comment (sndrs) [reply common/app/html/HtmlPage.scala]: ooooh – good spot!

Comment (sndrs) [reply common/app/html/HtmlPage.scala]: kind of surprised that compiled?

Comment (sndrs) [reply common/app/html/HtmlPage.scala]: ah, er, it didn't 🤦‍♂️

**Pull request merged**